nifi-toolkit/nifi-toolkit-cli/src/main/java/org/apache/nifi/toolkit/cli/impl/command/CommandOption.java

```diff
@@ -177,6 +177,10 @@ public enum CommandOption {
     USERNAME("usr", "username", "The username for authentication when obtaining an access token", true),
     PASSWORD("pwd", "password", "The password for authentication when obtaining an access token", true),
 
+    OIDC_URL("oidcurl", "oidcUrl", "The OIDC URL to access the token endpoint for the OAuth Client Credentials Flow", true),
```

Comment:
Regarding the property naming, it seems like it would be helpful to name this something like `oidcTokenUrl` to make it clear that this refers to the `token_endpoint`, not the OpenID Connect discovery configuration JSON. The description is clear, but making it clear in the property name would be helpful.